app/src/components/UI/Pages/LandingPage.tsx

```diff
@@ -14,9 +17,21 @@ export const LandingPage = () => {
           Version {config.version}, server commit ID {version}
         </p>
       </Container>
-      <Container className={styles.container}>
-        <Map />
-      </Container>
+      <Sidebar.Pushable as={Container} className={styles.container}>
+        <LandingPageSidebar setVisible={setVisible} visible={visible} />
+        <Sidebar.Pusher>
```

Comment:
I don't think we need a another button to show the sidebar as the search bar button will activate the sidebar.

Reply:
So the enter button on the search bar will toggle the sidebar on press.... and the map will be able to automatically show the vendors depending on which filters are selected? Just wanted to get this clear.

Reply:
This is what is written on the SDD:

> If the user decides to enter another destination, they can type in their location in the search bar provided. The results from that search will populate the left sidebar of the map, as well as show on the actual map (with a detailed popup).

So yes, the search results are shown on the sidebar and on the map.

Reply:
I think that having the option to toggle might make navigating the map easier, especially on small screens. I know that users may want to navigate even after results are shown. What do you think?

Reply:
On a mobile device, we wouldn't be showing a sidebar as people usually view their screens vertically.